package.json

```diff
@@ -33,6 +33,7 @@
   "devDependencies": {
     "@axe-core/puppeteer": "^4.1.1",
     "@babel/core": "^7.12.13",
+    "@department-of-veterans-affairs/formation": "^6.14.0",
```

Comment:
This allows us to get some of the familiar styles (fonts, etc.) in our Storybook environment.